exceptions/exceptions.hh

```diff
@@ -65,6 +65,7 @@ enum class exception_code : int32_t {
     WRITE_FAILURE   = 0x1500,
     READ_TIMEOUT    = 0x1200,
     READ_FAILURE    = 0x1300,
+    OVERFLOW_ERROR  = 0x1600,
```

Comment:
I see this in Cassandra source:

```
src/java/org/apache/cassandra/exceptions/ExceptionCode.java:    CDC_WRITE_FAILURE   (0x1600),
```

Let's allocate our errors in the range 0x1aff-0x1a01 (backwards) to minimize the chance of collision.

Reply:
sure

Reply:
@avi, why `0x1aff-1a01` rather than `0x1fff-0x1f01`?

Reply:
0x1fff is too special, somebody else might want to use it. 0x1aff is boring and no one would ever pick it on purpose.